mappings/net/minecraft/client/render/RenderLayer.mapping

```diff
@@ -115,8 +115,8 @@ CLASS net/minecraft/class_1921 net/minecraft/client/render/RenderLayer
 		ARG 0 texture
 	METHOD method_29380 getTranslucentMovingBlock ()Lnet/minecraft/class_1921;
 	METHOD method_29381 getItemPhaseData ()Lnet/minecraft/class_1921$class_4688;
-	METHOD method_29706 getGlintDirect ()Lnet/minecraft/class_1921;
-	METHOD method_29707 getEntityGlintDirect ()Lnet/minecraft/class_1921;
+	METHOD method_29706 getDirectGlint ()Lnet/minecraft/class_1921;
+	METHOD method_29707 getDirectEntityGlint ()Lnet/minecraft/class_1921;
```

Comment:
The getters should match the fields which are called `GLINT_DIRECT` and `ENTITY_GLINT_DIRECT`, which come from their IDs.

Reply:
Imo mojang used stupid naming there and we should not follow suit.

Reply:
What is the verdict? I changed it because the names were not fitting with our naming convention (they're stupid, as liach has said)

Reply:
Hmm, I'd say renaming is fine if the fields are also renamed to match the methods.

Reply:
I agree. Rename field and method